libflux/src/core/semantic/env.rs

```diff
@@ -135,4 +135,12 @@ impl Environment {
             None => panic!("cannot pop final stack frame from type environment"),
         }
     }
+
+    // Copy all the variable bindings from another Environment to the current environment.
+    // This does not change the current environment's parent and readwrite flag.
+    pub fn copy_bindings_from(&mut self, other: &Environment) {
```

Comment:
What is this used for?

Reply:
It is currently used to copy the variable binding we injected over to the prelude environment: https://github.com/influxdata/flux/blob/feat/option-reference-api/libflux/src/flux/lib.rs#L538

I made it a function because I think we might, in the future, need to track the references for more than one variable, and having such function will be useful (maybe overthinking).